### PR TITLE
Add Julia to Community Search Context Page

### PR DESCRIPTION
**Pull request by jdorfman**
## Test plan

My first PR. Need help 🙏

### CODE REVIEW

File: client/web/src/communitySearchContexts/routes.tsx [hunk 1]
```diff
@@ -64,4 +64,9 @@ export const communitySearchContextsRoutes: readonly LayoutRouteProps<any>[] = [
         render: props => <CncfCommunitySearchContextPage {...props} />,
         condition: ({ isSourcegraphDotCom }) => isSourcegraphDotCom,
     },
+    {
+        path: '/julia',
+        render: props => <JuliaCommunitySearchContextPage {...props} />,
```
Comment (novoselrok) [inline client/web/src/communitySearchContexts/routes.tsx:69]: You'll need to create the `JuliaCommunitySearchContextPage` component in the same directory as well. You can try copying the `Stanford.tsx` file and replacing all of the references to Stanford with Julia, and replacing the examples list with Julia examples. 

We can also hop on a call an I can take you through the process. I'm sorry we don't have a better way of doing this right now 🙁

Comment (jdorfman) [reply client/web/src/communitySearchContexts/routes.tsx:69]: @novoselrok forgot to add [it](https://github.com/sourcegraph-community/julia-context/blob/main/Julia.tsx). 🤦‍♂️ On it

Comment (jdorfman) [reply client/web/src/communitySearchContexts/routes.tsx:69]: @novoselrok I used the VSCode web version to add [Julia.tsx](https://github.com/jdorfman/sourcegraph/commit/b00ec0c5c2cc1bb49c8cfaa17c09f3d58242bd0a). I pushed the commit but it didn't update this PR.

**Comment by jdorfman** [comment]
Calling cla-bot to run the check again: @cla-bot check

**Comment by jdorfman** [comment]
Calling cla-bot to run the check again: @cla-bot check

**Comment by novoselrok** [comment]
Closed in favour of https://github.com/sourcegraph/sourcegraph/pull/32932

**Pull request closed without merging**